src/app/app.module.ts

```diff
@@ -40,10 +46,19 @@ import {MatButtonModule} from '@angular/material';
         CommonModule,
         MatCardModule,
         AppRoutingModule,
-        MatButtonModule
+        MatButtonModule,
+        MatSelectModule,
+        MatInputModule,
+        MatFormFieldModule,
+        MatSnackBarModule,
+        MatTabsModule
     ],
   providers: [],
   bootstrap: [ AppComponent ]
 })
 
+  // entryComponents: [
```

Comment:
Remember to keep your code clean by removing commented-out code. Code snippets such as this can be retrieved in future simply by checking past versions in Git.